src/components/pages/weather/elements/WeatherTempCell.js

```diff
@@ -2,15 +2,19 @@ import React, { Component } from 'react';
 
 export default class WeatherTempCell extends Component {
   render() {
-    const averageTemp = this.props.dayTemperatureData.avg;
-    const maxTemp = this.props.dayTemperatureData.max;
-    const minTemp = this.props.dayTemperatureData.min;
-    return (
+    const averageTemp = (this.props.dayTemperatureData.avg ? (this.props.dayTemperatureData.avg) : ('data unavailable'));
+    const maxTemp = (this.props.dayTemperatureData.max ? (this.props.dayTemperatureData.max) : ('data unavailable'));
+    const minTemp = (this.props.dayTemperatureData.min ? (this.props.dayTemperatureData.min) : ('data unavailable'));
+    return ((!this.props.dayTemperatureData.avg && !this.props.dayTemperatureData.max && !this.props.dayTemperatureData.min) ? (
+      <div className="temperature-cell">
+        <p>no temperature data available</p>
+      </div>
+    ) : (
       <div className="temperature-cell">
         <h4>Temperature: {averageTemp}</h4>
         <p>Max: {maxTemp}</p>
         <p>Min: {minTemp}</p>
       </div>
-    );
+    ));
   }
 }
```

Comment:
Similarly in this file in regards to:

- Using the **OR** operator (`||`)
- Trying to avoid the **NOT** operator (`!`)
- Thinking about which parentheses are necessary in our ternary operator.